### PR TITLE
completed dom-querying

### PR DESCRIPTION
**Pull request by salgarcia666**
sal garcia completed dom querying
Screenshot (154)

### CODE REVIEW

File: dom-querying/main.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+console.log('hello world');
+console.log(document.querySelector('h1'));
+console.dir('h1');
```
Comment (Zhiwutian) [inline dom-querying/main.js:3]: you need to save the elements in variables in order to use `dir` on them. Or, you have to pass the call to querySelector to the dir method.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
updates for the dir calls

File: dom-querying/main.js (hunk 1 above)
Comment (salgarcia666) [reply dom-querying/main.js]: Screenshot (155)

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**